### PR TITLE
CVE-2018-15610 update from Avaya CNA

### PR DESCRIPTION
**Pull request by AvayaSecurityAlerts**

### CODE REVIEW

**Comment by AvayaSecurityAlerts** [comment]
E-mail sent to cve@mitre.org with subject "Git Pilot participation for Avaya CNA"

File: 2018/15xxx/CVE-2018-15610.json [hunk 1]
```diff
@@ -1,18 +1,85 @@
 {
```
[the remark this answers is not in the document]
Comment (cve-team) [reply 2018/15xxx/CVE-2018-15610.json]: Also note that ASSIGNER in the JSON files should be an email address.

File: 2018/15xxx/CVE-2018-15610.json [hunk 2]
```diff
@@ -1,18 +1,85 @@
...
+               "product": {
+                  "product_data": [
+                     {
+                        "product_name": "IP Office",
+                        "version": {
+                           "version_data": [
+                              {
+                                 "affected": ">=",
+                                 "version_name": "9.1.2",
+                                 "version_value": "10.0"
```
Comment (cve-team) [inline 2018/15xxx/CVE-2018-15610.json]: The version information in the "affects" object does not seem to agree with the published advisory; eg, there's no mention about 9.x in the latter.

Comment (AvayaSecurityAlerts) [reply 2018/15xxx/CVE-2018-15610.json]: We don't include unsupported product versions in advisories, but the issue was originally reported in a 9.x load so we included it in the CVE.

**Comment by cve-team** [comment]
The second commit includes a file named ".gitignore".  Would you remove that?

File: 2018/15xxx/CVE-2018-15610.json (hunk 2 above)
Comment (cve-team) [reply 2018/15xxx/CVE-2018-15610.json]: I understand, but the info should be consistent across the two places in the JSON. Also, I'm not clear why you use two different attributes - "version_name" for the 9.x "load" and "version_value" for the 10.x one.

**Comment by AvayaSecurityAlerts** [comment]
Removed .gitignore, sorry about that. We don't use git much, so if we're doing anything improper or unusual please let us know.

File: 2018/15xxx/CVE-2018-15610.json (hunk 2 above)
Comment (AvayaSecurityAlerts) [reply 2018/15xxx/CVE-2018-15610.json]: I'm not sure either; it's the way the Vulnogram tool generated it. I changed it to be in line with other recent submissions.

**Comment by cve-team** [comment]
No worries -- everything looks good now.

**Pull request merged**